source/core/src/main/com/csse3200/game/components/ship/ShipTimeSkipComponent.java

```diff
@@ -0,0 +1,52 @@
+package com.csse3200.game.components.ship;
+
+import com.csse3200.game.components.Component;
+import com.csse3200.game.components.ship.ShipProgressComponent.Feature;
+import com.csse3200.game.services.ServiceLocator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Set;
+
+import static com.csse3200.game.services.TimeService.MORNING_HOUR;
```

Comment:
`TimeService` info shouldn't be in this component as unnecessary coupling

Reply:
Removed as part of the timescale update